src/styles/_custom.scss

```diff
@@ -304,3 +304,40 @@ div.is-rounded img {
     }
   }
 }
+
+.jf-cookies-banner {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  border-top: 1px solid $justfix-black;
+
+  img {
+    filter: invert(1) contrast(0.7);
+  }
+
+  .link {
+    display: inline;
+  }
+
+  .button {
+    width: 14px;
+    height: 14px;
+  }
```

Comment:
I don't think this is needed? On my end all the text is squeezed 
<img width="203" alt="image" src="https://user-images.githubusercontent.com/16906516/180493634-e2707e03-14fb-4bcf-93d0-51b52eb059bd.png">

Reply:
oops!